src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -52,13 +52,12 @@
 
 .unified-share-modal {
     /** title and classifiction label */
-    .bdl-UnifiedShareModalTitle {
-        display: flex;
-        flex-flow: row wrap;
+    .bdl-UnifiedShareModalTitle-classification {
+        display: inline;
+    }
 
-        .bdl-UnifiedShareModalTitle-classification {
-            margin-left: 6px;
-        }
+    .bdl-UnifiedShareModalTitle {
+        margin-right: 6px;
```

Comment:
this should probably be on the grid `bdl-grid-unit` (*2?)

Reply:
Thanks for the comment! For classification badges we have the space of 6px on other places such as page header/ share link settings modal. `bdl-grid-unit` is 4px. I think we should keep the space consistent as others. 
![Screen Shot 2020-08-13 at 5 01 37 PM](https://user-images.githubusercontent.com/20669641/90198490-af064000-dd86-11ea-9d88-c43c729e871e.png)
![Screen Shot 2020-08-13 at 5 01 26 PM](https://user-images.githubusercontent.com/20669641/90198494-b1689a00-dd86-11ea-8960-0cf829c1d656.png)

Reply:
if so, that means these ones are inconsistent with the other spacings used in the app? if all other ones match at 6px, i suppose they should be internally consistent, but I think design will have to make the final call on that since they created the grid for us to use